### PR TITLE
Get batch by id jayant

### PR DESCRIPTION
**Pull request by tavisca-jkanathe**

### CODE REVIEW

File: src/Tavisca.TMS.Persistence/Translator/Batch/BatchTranslator.cs [hunk 1]
```diff
@@ -16,16 +17,18 @@ public BatchDetails Translate(BatchDto BatchDto)
                 Id = BatchDto.Id,
                 Name = BatchDto.Name,
                 Status = BatchDto.IsActive==true ? "Active":"InActive",
-                count=BatchDto.count
+                count=BatchDto.Count
             };
         }
-        public GetBatchByIdDetails TranslateToGetBatchByIdDetails(BatchDto BatchDto)
+        public GetBatchByIdDetails TranslateToGetBatchByIdDetails(BatchDto BatchDto, List<BatchEmployeeDetails> batchEmployeeDetails)
```
[the remark this answers is not in the document]
Comment (tavisca-jkanathe) [reply src/Tavisca.TMS.Persistence/Translator/Batch/BatchTranslator.cs]: done

File: src/Tavisca.TMS.Web/Controllers/UserLoginController.cs [hunk 2]
```diff
@@ -12,6 +14,8 @@ namespace Tavisca.TMS.Web.Controllers
 {
     [Route("api/[controller]")]
     [ApiController]
+
```
[the remark this answers is not in the document]
Comment (tavisca-jkanathe) [reply src/Tavisca.TMS.Web/Controllers/UserLoginController.cs]: done

File: src/Tavisca.TMS.Persistence/Dal/BatchDal.cs [hunk 3]
```diff
@@ -314,79 +320,39 @@ public async Task<BatchDto> GetBatchById(string batchId)
...
             }
         }
-            
-            
 
-        
-        
-        private  BatchDto ExtractDto(DbDataReader reader)
+
+        public BatchDto FetchBatchDetails(DataRowCollection BatchDatadataRowCollection, DataRowCollection BatchEmployeesdataRowCollection)
```
[the remark this answers is not in the document]
Comment (tavisca-jkanathe) [reply src/Tavisca.TMS.Persistence/Dal/BatchDal.cs]: done

File: src/Tavisca.TMS.Tests/Data/Dal/BatchDalFixture.cs [hunk 4]
```diff
@@ -72,16 +72,7 @@ public async Task BatchDal__GetEmployeesIdOfBatchList_should_handle_exception_th
...
-
-            string batchIdThatDoesntExist = "51377af6-f836-4385-b8a5-76ff2bf1681e";
-            _mySqlConnection.Setup(x => x.GetConnection()).Returns(_connection);
-
-            await Assert.ThrowsAsync<RowNotInTableException>(() => _batchDal.GetBatchById(batchIdThatDoesntExist));
-        }
-
+       
         //[Fact]
         //public void BatchDal__Bad_connection_GetConnection_should_throw_an_error_that_will_be_handled()
```
Comment (tavisca-balaji) [inline src/Tavisca.TMS.Tests/Data/Dal/BatchDalFixture.cs]: remove commented code

**Pull request merged**